apps/base-docs/docs/connecting-to-the-blockchain/overview-of-providers.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Overview of Providers
+description: Learn what providers are and why you need one.
+hide_table_of_contents: false
+---
+
+Onchain apps need frontends, sometimes called dApps, to enable your users to interact with your smart contracts. A _provider_ makes the connection from frontend to blockchain, and is used to read data and send transactions.
+
+---
+
+## Prerequisites
+
+Before this lesson, you should:
+
+- Be familiar with modern, frontend web development
+- Possess a general understanding of the EVM and smart contracts
+- Ideally, understand how to build a frontend with [Next.js]
+
+---
+
+## Objectives
+
+By the end of this lesson you should be able to:
+
+- Compare and contrast public providers vs. vendor providers vs. wallet providers
+- Select the appropriate provider for several use cases
+
+---
+
+## Types of Providers
+
+In blockchain development, the term _provider_ describes a company or service that provides an API enabling access to the blockchain as a service. This is distinct from the provider in the the React Context API or Redux Provider, though you will also wrap your app with `wagmiConfig` in a similar manner.
```

Comment:
Last sentence can be consolidated down to just mention that the word `Provider` in the blockchain context is different from `Provider` in the React.js development context. Mentioning specific React.js providers (WagmiProvider, ReduxProvider, ThemeProvider, etc.) is probably not necessary.

Reply:
Got rid of the specific reference to wagmi per your other suggestion, and reworded.  I do think it's important here to be a little more specific since a big part of the audience appears to not be the most expert in React in general.  Including me.